test/evergreen/run_format_configs.sh

```diff
@@ -63,17 +62,19 @@ wait_for_process()
 				fi
 
 				echo "Exit status of config ${config_name} is ${exit_status}"
-				break
```

Comment:
I'd prefer to call `continue` explicitly here even though the effect is the same right now. It's a peace of mind in case some new lines are added after the if block later on.

Reply:
Fixed in e86d6151fdda76661610cd6736e283617419375b